app/controllers/shipit/api/stacks_controller.rb

```diff
@@ -16,6 +23,10 @@ def show
       def stack
         @stack ||= stacks.from_param!(params[:id])
       end
+
+      def create_params
```

Comment:
For the API shipit uses explicit-parameters. You can look at other API controllers or webhook controller.

Also for creation parame shouldn't be nested under :stack